src/AutoRest.CSharp/readme.md

```diff
@@ -18,13 +18,20 @@ output-scope:
 shared-source-folders: $(this-folder)/Generator.Shared;$(this-folder)/Azure.Core.Shared
 ```
 
-```yaml !$(data-plane)
+```yaml !$(data-plane) && !$(azure-arm)
 modelerfour:
   flatten-models: true
   flatten-payloads: true
   group-parameters: true
 ```
 
+```yaml $(azure-arm)
+modelerfour:
+  flatten-models: true
```

Comment:
Lets get a diff in azure-sdk-for-net on what these flags on/off will result in to see if the changes make sense.

Reply:
Did we ever get that PR?  Can you link it here?